src/DocBlock/Tags/Method.php

```diff
@@ -261,10 +261,18 @@ public function __toString(): string
     {
         $arguments = [];
         foreach ($this->parameters as $parameter) {
+            $parameterDefaultValueStr = null;
+            if ($parameter->getDefaultValue() !== null) {
+                $parameterDefaultValueStr = $parameter->getDefaultValue();
+                settype($parameterDefaultValueStr, (string)$parameter->getType());
```

Comment:
This doesn't seem to be correct to me. Default values can be complex definitions. 
It might be good to check with a number of testcases to ensure this works as expected. 

Please do also add a testcase with a custom class.

Reply:
You mean isolate the test in a custom class ?

Reply:
Oh no i understood, give me some time 😄